utilities/logger_utilities.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import os
+from datetime import datetime
+import pathlib
+
+def setup_logger():
```

Comment:
Will move hard coded stuff like `days_old` to config file once we create folder for configurations